.github/workflows/zero-downtime-tests.yml

```
@@ -325,11 +334,15 @@ jobs:
           # Optional: pass GITHUB_TOKEN to avoid rate limiting.
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Install npm
+        id: install_npm
```

please rename this step to make it a little bit clearer. (you can also remove the ID if it's not used somewhere else in the pipeline)
```suggestion
      - name: Install Node dependencies
```